error: Could not connect to the database. R is not defined {"stack":"ReferenceError: R is not defined\n    at _callee$ (D:\\CLI_CLASS\\FandooNotes_webAPI\\src\\config\\/database.js:10:32)\n    at tryCatch (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:63:40)\n    at Generator.invoke [as _invoke] (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:294:22)\n    at Generator.next (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:119:21)\n    at asyncGeneratorStep (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:32:7\n    at new Promise (<anonymous>)\n    at D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:21:12\n    at database (D:\\CLI_CLASS\\FandooNotes_webAPI\\src\\config\\/database.js:4:15)","timestamp":"2022-01-11T11:38:59.339Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T11:38:59.396Z"}
error: Could not connect to the database. R is not defined {"stack":"ReferenceError: R is not defined\n    at _callee$ (D:\\CLI_CLASS\\FandooNotes_webAPI\\src\\config\\/database.js:10:32)\n    at tryCatch (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:63:40)\n    at Generator.invoke [as _invoke] (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:294:22)\n    at Generator.next (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\regenerator-runtime\\runtime.js:119:21)\n    at asyncGeneratorStep (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:32:7\n    at new Promise (<anonymous>)\n    at D:\\CLI_CLASS\\FandooNotes_webAPI\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:21:12\n    at database (D:\\CLI_CLASS\\FandooNotes_webAPI\\src\\config\\/database.js:4:15)","timestamp":"2022-01-11T11:42:37.924Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T11:42:37.932Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T11:43:47.321Z"}
info: Connected to the database. {"timestamp":"2022-01-11T11:43:47.348Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T11:55:01.510Z"}
info: Connected to the database. {"timestamp":"2022-01-11T11:55:01.535Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T11:55:24.285Z"}
info: Connected to the database. {"timestamp":"2022-01-11T11:55:24.307Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: FondooNote_webAPI.users index: Email_1 dup key: { Email: "vishal@gamail.com" } 
      url - /api/v1/register 
      method - POST 
      IP - ::1
     {"timestamp":"2022-01-11T12:02:31.779Z"}
error: 
    status - 500 
    message - RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: 11000
    at ServerResponse.writeHead (_http_server.js:259:11)
    at ServerResponse.writeHead (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (_http_server.js:250:8)
    at write_ (_http_outgoing.js:711:9)
    at ServerResponse.end (_http_outgoing.js:825:5)
    at ServerResponse.send (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\express\lib\response.js:221:10)
    at ServerResponse.json (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\express\lib\response.js:267:15)
    at appErrorHandler (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\express\lib\router\index.js:321:13) 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T12:02:31.837Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T12:30:54.829Z"}
info: Connected to the database. {"timestamp":"2022-01-11T12:30:54.906Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T14:57:16.982Z"}
info: Connected to the database. {"timestamp":"2022-01-11T14:57:17.084Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:25:26.896Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:25:26.924Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:27:36.144Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:27:36.183Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/register/addnote 
      method - POST 
      IP - ::1
     {"timestamp":"2022-01-11T16:27:47.177Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T16:28:00.768Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:30:06.304Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:30:06.334Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:31:24.076Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:31:24.131Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:31:41.066Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:31:41.114Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:32:50.439Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:32:50.488Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:24:32)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T16:32:50.562Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:33:26.616Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:33:26.672Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:33:49.327Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:33:49.357Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:34:45.954Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:34:46.000Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:35:42.519Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:35:42.571Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:36:02.616Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:36:02.657Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:41:20.855Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:41:20.877Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:41:46.733Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:41:46.789Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:42:32.074Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:42:32.098Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:46:30.590Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:46:30.660Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:49:39.160Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:49:39.201Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:49:48.060Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:49:48.116Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:51:40.837Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:51:40.873Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:52:13.732Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:52:13.767Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:53:02.776Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:53:02.817Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T16:53:41.520Z"}
info: Connected to the database. {"timestamp":"2022-01-11T16:53:41.542Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:03:00.743Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:03:00.811Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:03:31.760Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:03:31.782Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:19:35.658Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:19:35.680Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:20:28.904Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:20:28.926Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:25:40.155Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:25:40.177Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:25:51.099Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:25:51.122Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:39:54.855Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:39:54.909Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:45:08.928Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:45:09.008Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:46:04.757Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:46:04.780Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:47:07.062Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:47:07.103Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:47:26.974Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:47:27.002Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:48:21.873Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:48:21.912Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:48:30.050Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:48:30.092Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T17:48:30.322Z"}
error: 
      status - 400
      message - Authorization token incorrect 
      url - /api/v1/register/addnote 
      method - POST 
      IP - ::1
     {"timestamp":"2022-01-11T17:48:37.652Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/register/addnote 
      method - POST 
      IP - ::1
     {"timestamp":"2022-01-11T17:48:48.330Z"}
error: 
      status - 400
      message - Authorization token incorrect 
      url - /api/v1/register/addnote 
      method - POST 
      IP - ::1
     {"timestamp":"2022-01-11T17:48:59.019Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:49:04.181Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:49:15.443Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:49:59.653Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:49:59.690Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:50:39.807Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:50:39.870Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:50:48.384Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:53:23.710Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:53:32.979Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:53:33.040Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:53:42.970Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:53:54.034Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:53:59.685Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:54:01.255Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:54:02.114Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:55:57.851Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:55:57.878Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:56:04.976Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:57:01.183Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:57:01.210Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T17:57:02.252Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T17:58:15.009Z"}
info: Connected to the database. {"timestamp":"2022-01-11T17:58:15.069Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T17:58:15.583Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'Email' of null
    at _callee2$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:32:23)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:01:08.063Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:03:13.263Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:03:13.293Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:05:06.753Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:08:31.161Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:08:31.368Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token } in JSON at position 192
    at JSON.parse (<anonymous>)
    at parse (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\body-parser\lib\types\json.js:89:19)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:224:16)
    at done (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at IncomingMessage.emit (domain.js:470:12)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:09:10.940Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token } in JSON at position 192
    at JSON.parse (<anonymous>)
    at parse (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\body-parser\lib\types\json.js:89:19)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:224:16)
    at done (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at IncomingMessage.emit (domain.js:470:12)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:09:23.426Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:10:20.536Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:10:58.848Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:11:27.170Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:11:27.238Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T18:11:27.612Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:11:29.463Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:11:30.807Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee3$ (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:49:9)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.addNote (D:\CLI_CLASS\FandooNotes_webAPI\src\services\/user.service.js:48:21) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:11:31.341Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:12:02.287Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:12:02.354Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:12:46.679Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:12:46.711Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:12:48.127Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:13:30.342Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:13:30.371Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:13:38.250Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:14:07.166Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:14:07.206Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:14:34.469Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:14:34.492Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:14:35.561Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:14:42.471Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:15:15.791Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:15:15.835Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:15:25.761Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:15:25.796Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T18:15:26.522Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:00.980Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:07.310Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:21.089Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:26.852Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:41.693Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:46.348Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:17:54.535Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:18:13.657Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:18:19.726Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:18:42.732Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:18:42.791Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:18:44.089Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:19:02.501Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:19:19.990Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:19:29.833Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:20:06.568Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:20:06.593Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:12.128Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:13.988Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:15.117Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:15.493Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:15.886Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:20:25.129Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:21:20.717Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:21:20.744Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-11T18:21:21.053Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:21:49.470Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:21:49.514Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:21:51.164Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:22:07.711Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:22:10.891Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:22:24.207Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:22:25.675Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:23:31.297Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:23:31.326Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'user' of undefined
    at _callee$ (D:\CLI_CLASS\FandooNotes_webAPI\src\middlewares\/auth.middleware.js:25:13)
    at tryCatch (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\CLI_CLASS\FandooNotes_webAPI\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/register/addnote 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-11T18:23:36.194Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:24:33.853Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:24:33.900Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:25:34.224Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:25:34.262Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:25:42.071Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:25:42.098Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:26:05.610Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:26:05.721Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:26:42.778Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:26:42.817Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:28:27.819Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:28:27.860Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:29:12.745Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:29:12.767Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:29:25.316Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:29:25.338Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-11T18:29:37.469Z"}
info: Connected to the database. {"timestamp":"2022-01-11T18:29:37.490Z"}
